lib/alterUI.js

```diff
@@ -86,8 +87,19 @@ define(function (require, exports, module) {
         $(".content").css("right","0");
     }
 
+    /**
+     * Used to add a button to the status bar for the
+     * detached live preview.
+     */
+    function addLivePreviewButton() {
+        $("#status-indicators").append("<button id=liveDevButton>Live Preview</button>");
```

Comment:
I really think this needs to be an icon, not a button with that much text.  Can you look at other Brackets extensions that do similar things and just do an icon or img?